java/operator/src/main/java/com/instaclustr/cassandra/operator/service/BackupControllerService.java

```diff
@@ -128,9 +137,16 @@ private void createOrReplaceBackup(final Backup backup) throws ApiException, Unk
                 .parallelStream()
                 .map(x -> callBackupApi(x, backup))
                 .anyMatch(e -> !e)) {
-            //TODO: Set backup status to PROCESSED
+            BackupSpec backupSpec = backup.getSpec();
+            backupSpec.setStatus("ERROR");
+            backup.setSpec(backupSpec);
+            customObjectsApi.patchNamespacedCustomObject("monitoring.coreos.com", "v1", backup.getMetadata().getNamespace(), "cassandra-backups", backup.getMetadata().getName(), backup);
+        } else {
+            BackupSpec backupSpec = backup.getSpec();
```

Comment:
I think these two if branches can be consolidated. The only difference seems to be the status value -- "ERROR" vs "PROCESSED"?

Reply:
Consolidated